### ISSUE TITLE
A desktop app that automatically tracks activity

### ISSUE THREAD

**Issue by almarklein** [almarklein/timetagger#45]
... and sends it to a TimeTagger service (either at timetagger.app or self-hosted).

See e.g. ActivityWatch

**Comment by dzmitry-lahoda** [comment]
i will not work, tried. too tied to platform and apps. only can imagine how to have API to sink data into timetagger, but not trying to impl. imagine better allowance tools like vscode plugins or ms office or any specific exact tool to put data into time tagger. do not try to track on own, that is too wild.

**Comment by almarklein** [comment]
Yeah, I think a better way would be to use ActivityWatch and push the data to a timetagger server.

**Comment by dzmitry-lahoda** [comment]
another integration, todo apps and project tools. so I tag these with some tags. and than when start to do them, i send request to time tagger. so it is more sturtured approach. same can work for training apps, like tagging specific poses/muscles.
